applications/CompressiblePotentialFlowApplication/python_scripts/define_wake_process_3d.py

```diff
@@ -30,56 +15,37 @@ def __init__(self, Model, settings):
 
         # Check default settings
         default_settings = KratosMultiphysics.Parameters(r'''{
-            "model_part_name": "",
-            "body_model_part_name": "",
-            "wake_stl_file_name" : "",
-            "output_wake": false,
+            "trailing_edge_model_part_name" : "",
+            "body_model_part_name"          : "",
+            "check_wake_condition_tolerance": 1e-1,
             "wake_process_cpp_parameters":    {
```

Comment:
Why two levels? The Python and cpp settings must be the same (another reason for my comment above). As we are changing the settings (i.e., breaking the retrocompatibility), I'd take the chance to make it properly.